src/lib.rs

```diff
@@ -0,0 +1,108 @@
+pub mod config {
+    use serde::Deserialize;
+
+    use std::error::Error;
+    use std::fs::File;
+    use std::io::BufReader;
+    use std::path::Path;
+
+    #[derive(Deserialize, Debug)]
+    pub struct Config {
+        pub db_address: String,
+    }
+
+    pub fn read_json_config(path: impl AsRef<Path>) -> Result<Config, Box<dyn Error>> {
+        let file = File::open(path)?;
+        let reader = BufReader::new(file);
+
+        // read json from file
+        let json_data = serde_json::from_reader(reader)?;
+
+        Ok(json_data)
+    }
+}
+
+pub mod db {
+    use chrono::{serde::ts_seconds, serde::ts_seconds_option, DateTime, Utc};
+    use mongodb::{bson::doc, sync::Collection};
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Debug, Serialize, Deserialize)]
+    // Task型
+    pub struct Task {
+        #[serde(with = "ts_seconds_option")]
+        pub scheduled_date: Option<chrono::DateTime<Utc>>,
+        #[serde(with = "ts_seconds")]
+        pub register_date: DateTime<Utc>,
```

Comment:
どちらかに揃えた方が良いのでは, 一応後者( `::register_date` )に合わせてsuggest

```suggestion
        #[serde(with = "ts_seconds_option")]
        pub scheduled_date: Option<DateTime<Utc>>,
        #[serde(with = "ts_seconds")]
        pub register_date: DateTime<Utc>,
```

Reply:
acc18947d647274631f6860e89d9fa24db41b25d
done